### ISSUE TITLE
group profile for transactional db workload

### ISSUE THREAD

**Issue by atinmu** [gluster/glusterd2#1250]
group profile infrastructure already exists in GD2, however we'd need to have a profile for db workload tunings to be included for GD2 which should be as follows:

performance.open-behind=on
performance.write-behind=off
performance.stat-prefetch=off
performance.quick-read=off
performance.strict-o-direct=on
performance.read-ahead=off
performance.io-cache=off
performance.readdir-ahead=off
performance.client-io-threads=on
server.event-threads=4
client.event-threads=4
performance.read-after-open=yes

### Observed behavior
No db profile is currently present

### Expected/desired behavior
DB profile should be present

### Information about the environment:

- Glusterd2 version used (e.g. v4.1.0 or master): 
- Operating system used: 
- Glusterd2 compiled from sources, as a package (rpm/deb), or container: 
- Using External ETCD: (yes/no, if yes ETCD version):
- If container, which container image: 
- Using kubernetes, openshift, or direct install: 
- If kubernetes/openshift, is gluster running inside kubernetes/openshift or outside: 

### Other useful information

- glusterd2 config files from all nodes (default /etc/glusterd2/glusterd2.toml)
- glusterd2 log files from all nodes (default /var/log/glusterd2/glusterd2.log)
- ETCD configuration
- Contents of `uuid.toml` from all nodes (default /var/lib/glusterd2/uuid.toml)
- Output of `statedump` from any one of the node

### Useful commands

- To get glusterd2 version
    ```
    glusterd2 --version
    ```
- To get ETCD version
    ```
    etcd --version
    ```
- To get output of statedump
    ```
    curl http://glusterd2-IP:glusterd2-Port/statedump
    ```

**Comment by prashanthpai** [comment]
Some option names are legacy, they have to be migrated to new names. We also need https://github.com/gluster/glusterd2/issues/376 to be finalised. Currently, there's no well-defined mechanism in volgen to add or remove a xlator from graph.

**Comment by aravindavk** [comment]
moving this to `GCS/0.3` since this depends on volgen template support PR

**Comment by atinmu** [comment]
@aravindavk Now that volgen template support PR is in, can we focus on getting this closed too?

**Comment by atinmu** [comment]
@aravindavk ping! GCS/0.3 is tomorrow and we need to decide if this is getting addressed or moved to GCS/0.4?
